packages/react/src/components/Modal/Modal.tsx

```diff
@@ -495,7 +670,7 @@ Modal.propTypes = {
    * Specify CSS selectors that match DOM elements working as floating menus.
    * Focusing on those elements won't trigger "focus-wrap" behavior
    */
-  selectorsFloatingMenus: PropTypes.arrayOf(PropTypes.string),
+  selectorsFloatingMenus: PropTypes.arrayOf(PropTypes.string.isRequired),
```

Comment:
Can we keep this as not required? In https://github.com/carbon-design-system/carbon/pull/14720 we're removing all default props and this will throw console warnings (even if we provide a default function parameter)

Reply:
@tw15egan the `isRequired` only pertains to the `string` part, not to the array / prop itself. It basically disallows nullish values inside the array, but the array itself can be `undefined`. This is to match the TS type constraint: `selectorsFloatingMenus?: string[]`. Otherwise, we'd need to type it as `selectorsFloatingMenus?: (string | null | undefined)[]` and handle nullish values in the code (right now we don't). I think it's not worth it.

Reply:
Oh right, thanks for clarifying! This works great then.